documentContentProviders/dockerInspect.ts

```diff
@@ -0,0 +1,22 @@
+import { TextDocumentContentProvider, Uri, window, workspace } from "vscode";
+import { docker } from "../commands/utils/docker-endpoint";
+
+export const IMAGE_DOMAIN = "image";
+export const SCHEME = "docker-inspect";
+export const URI_EXTENSION = ".json";
```

Comment:
@chrisdias I removed it because the watch (which obviously doesn't exit) was preventing the debugger from starting. That said, maybe I was doing something wrong, so I could revert this change.